x-pack/plugins/security_solution/public/management/cypress/screens/stack_management/role_page.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loadPage } from '../../tasks/common';
+
+/**
+ * Navigates to the Stack Management security role page. If `roleName` argument is passed,
+ * then the edit page for that role is displayed. Otherwise the "create new" role form page
+ * will be loaded.
+ *
+ * @param roleName
+ */
+export const navigateToRolePage = (roleName?: string): void => {
+  return loadPage(`/app/management/security/roles/edit${roleName ? `/${roleName}` : ''}`);
```

Comment:
> having some issues with getting the tests to run in serverless, but code is ready for review.

i think this url does not exist on serverless. i'm not sure how to test this on serverless - isn't role setting outside of kibana?

Reply:
FYI - I figured out why the role management was not showing up for our tests. Custom roles for serverless are still behind a feature flag, but already enabled in env. created from cloud QA, thus why I was seeing them.
I have not enabled them for our Cypress tests by default.

Reply:
nice, thanks!